app/routes/country.js

```diff
@@ -39,29 +44,45 @@ function createModelNotEligible () {
   }
 }
 
+function getPostCodeInput(postcodeData) {
+  let postcode = postcodeData && postcodeData !== null ? postcodeData : ' ' 
+  return `<label class="govuk-label" for="project_postcode">
+  Enter Postcode</label><input class="govuk-input govuk-!-width-one-third" id="project_postcode" name="project_postcode" value=${postcode}>`
+  
+}
+
 module.exports = [
   {
     method: 'GET',
     path: '/country',
-    handler: (request, h) => h.view('country', createModel(null))
+    handler: (request, h) => {
+      const inEngland = request.yar.get('inEngland') || null
+      const postcodeData = inEngland !== null ? request.yar.get('project_postcode') : null
+
+      return h.view('country', createModel(null, inEngland, postcodeData))
```

Comment:
instead of sending postcodeData - will still be processed in createModel by the function getPostCodeInput() - why not do this:

handler: (request, h) => {
  const inEngland = ...
  const postcodeData = ...
  const postcodeInput = getPostCodeInput (postcodeData)

  return h.view('country', createModel(null, inEngland, getPostCodeInput))

}

and then in createModel( errorMessage, data, postcodeInput ){
  ...
  conditional: { html: postcodeInput }
  ...
}